python/tools/build-wheels.sh

```diff
@@ -6,15 +6,15 @@ function repair_wheel {
     if ! auditwheel show "$wheel"; then
         echo "Skipping non-platform wheel $wheel"
     else
-        auditwheel repair "$wheel" --plat "manylinux2010_x86_64" -w /io/python/wheels/
+        auditwheel repair "$wheel" --plat "manylinux2014_x86_64" -w /io/python/wheels/
```

Comment:
Where does the build happen? Asking as the Docker image used to do the build would need to be updated as well. `manylinux2010` is CentOS 6 based whereas `manylinux2014` is CentOS 7 based

Reply:
Thanks  --  currently the builds are happening locally. FWIW, I run Ubuntu 20.04, so the newer Docker image should be no problem.

Reply:
Sorry was more confused by the fact this script is called `build-wheels.sh`, but doesn't seem to include a build step

Ok though the build is happening in a `manylinux` Docker image, right?

Reply:
Correct. Specifically I was going to use https://github.com/pandas-dev/pandas2/issues/10